src/barril/units/_array.py

```diff
@@ -243,10 +243,18 @@ def __getitem__(self, index):
     def __iter__(self):
         return iter(self.values)
 
+    def _calculate_floor_division(self, other, div_or_rdiv_function):
```

Comment:
Barril uses CamelCase right?

Reply:
True, I was around those `__xx__` methods and forgot that.

Reply:
Anyway, I've refactor that code to use an approach that is more consistent with other operations, i.e., implement the operation in `unit_database` and use it from there.